### DISCUSSION TITLE
[Bug] FF7 crash during the Lifesteam Flashback scene

### DISCUSSION THREAD

**Discussion by Aynath** [julianxhokaxhiu/FFNx#437] [General]
Hi,

The game always crashes at the moment when little Cloud talks to Tifa about the day when her mother died.
![image](https://user-images.githubusercontent.com/19351933/163725788-9567fe9c-7b56-4f19-9e97-b134fff669a3.png)

The crash occurs right after that scene the screen capture shows. 
It should show the scene where Tifa has fallen with Cloud on a white background. The error message was a generic unhandled exception. Here's the log : [FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/8502112/FFNx.log)
Fortunately, I play the NT mod, which let me to skip this part. I first thought it was due to the 60 FPS mod but even when disabled the crash occurs. Seems like there's a file missing according to the log.

**Comment by julianxhokaxhiu** [comment]
Thanks for the report. As you're using 7th, no support is given here as the crash is not reproducible in FFNx Vanilla. Consider approaching for help on Discord.
